dtmi/stmicroelectronics/steval_proteus1/standard_if-1.json

```diff
@@ -0,0 +1,301 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:stmicroelectronics:steval_proteus1:standard_if;1",
+  "@type": "Interface",
+  "description": {
+    "en": "Reports temperature, accelerometer, gyroscope, predictive speed status, predictive acceleration peak status, predictive frequency domain status, battery status"
+  },
+  "displayName": {
+    "en": "Standard interface for PROTEUS"
+  },
+  "contents": [
+    {
+      "@type": "Telemetry",
```

Comment:
could use co-type "Unitless" and unit "percent"

Reply:
tried but fails validation

Reply:
My bad.  DTDL does not have a generic Unitless semantic type, only specific types that are unitless, such as RelativeHumidity.  It also does not have a BatteryLevel or similarly appropriate type.